pkgs/by-name/ll/llama-cpp/package.nix

```
@@ -105,15 +111,26 @@ effectiveStdenv.mkDerivation (finalAttrs: {
   ++ lib.optionals blasSupport [
     "-DLLAMA_BLAS=ON"
     "-DLLAMA_BLAS_VENDOR=OpenBLAS"
+  ]
+  ++ lib.optionals (!static) [
+    (lib.cmakeBool "BUILD_SHARED_LIBS" true)
   ];
 
   installPhase = ''
     runHook preInstall
 
     mkdir -p $out/bin
+    mkdir $out/lib
+    ${lib.optionalString (!static) ''
```

```suggestion
    ${lib.optionalString (!static) ''
      mkdir $out/lib
```

Otherwise it's just an empty directory for no reason.

---

nice catch